Seth-Daily/Seth-Daily.md

```diff
@@ -13,12 +13,14 @@ quoteAuthor: "Mark Twain"
 
 [Editing your profile]: https://github.com/SSWConsulting/People/wiki/3.-Editing-your-profile
 
-Seth is an SSW Marketing Assistant and Account Manager. He has a particular interest in ChatGPT and other artificial intelligence tools, and their applications for content creation and  enhancing productivity in the workplace.
-
+Seth is an SSW Marketing Assistant and Account Manager. He has an interest in artificial intelligence and its applications for creativity and productivity. He's always making the best prompts for ChatGPT and Midjourney and stays up to date on the latest AI techniques. He likes contributing to [SSW Rules](https://www.ssw.com.au/rules/) because the ultimate goal is optimization of everything, and in pursuit of that masterpiece, best practices are the most advanced blueprint that we have.
 
+He also contributes in other ways, from marketing initiatives to organizing [events](https://www.ssw.com.au/ssw/Events/?tech=all&type=all) and learning account management skills under the guidance of [Uly](https://www.ssw.com.au/people/uly/).
```

Comment:
Organizing events is also part of marketing. Consider changing it 
from
> He also contributes in other ways, from marketing initiatives to organizing [events](https://www.ssw.com.au/ssw/Events/?tech=all&type=all)
to
He also contributes in other ways, from helping run [events](https://www.ssw.com.au/ssw/Events/?tech=all&type=all) to implementing AI in Marketing where possible